packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -97,10 +97,8 @@ export class SpanImpl implements Span {
   private readonly _startTimeProvided: boolean;
 
   /**
-   * Constructs a new Span instance.
-   *
-   * @deprecated calling Span constructor directly is not supported. Please use tracer.startSpan.
```

Comment:
note to reviewer: this is a leftover from #5048. The class is now internal and used in `Tracer` class